carbon-user-operation-eventListener-archetype/src/main/resources/archetype-resources/src/main/java/__listener-name__UserOperationEventListener.java

```diff
@@ -0,0 +1,347 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package ${package};
+
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.context.CarbonContext;
+import org.wso2.carbon.user.api.Permission;
+import org.wso2.carbon.user.core.UserStoreException;
+import org.wso2.carbon.user.core.UserStoreManager;
+import org.wso2.carbon.user.core.common.AbstractUserOperationEventListener;
+
+import java.util.List;
+import java.util.Map;
+
+
+
+public class ${listener-name}UserOperationEventListener extends AbstractUserOperationEventListener {
+
+    //Sample Code
+    private static Log audit = LogFactory.getLog(${listener-name}UserOperationEventListener.class);
+    private static String AUDIT_MESSAGE = "Initiator: %s performed the Action: %s on Target: %s ";
+
+    @Override
+    public int getExecutionOrderId() {
+
```

Comment:
```suggestion
 
 //Todo  implementation required
```

Reply:
DO this for all the places

Reply:
mandatory places -->  implementation is required.
other places --> . implement if required